include/classes/Language/LanguageEditor.class.php

```diff
@@ -119,11 +115,11 @@ function checkFilter($param) {
 
 	// public
 	function printTerms($terms){
-		global $addslashes, $languageManager; // why won't $addslashes = $this->addslashes; work?
+		global $languageManager; // why won't $this->addSlashes work?
```

Comment:
Please remove the code comment that no longer makes sense.

Reply:
Done